griptape-aws-bill-pdf-to-csv/README.md

```diff
@@ -31,20 +25,31 @@ This sample converts an AWS bill PDF into a CSV. This is most useful for custome
 You can run this Structure locally by providing the following inputs:
 
 ```
-python structure.py -s <source_s3_uri> -d <destination_s3_uri>
+python structure.py -b <bucket_id> -d <workdir> -p <pdf_asset_file_name> -c <csv_asset_file_name>
+```
+
+You may also run the `upload.py` and `download.py` scripts locally to upload and download files directly:
+```
+python upload.py -b <bucket_id> -p <local_file_path> -d <workdir> -n <asset_file_name>
+
+python download.py -b <bucket_id> -p <local_file_path> -d <workdir> -n <asset_file_name>
 ```
 
 ### Griptape Cloud
 
 You can create a run via the API or the UI. When creating runs in the UI, you will need to specify parameters on their own lines.
 
 ```
--s
-<source_s3_uri>
+-b
+<bucket_id>
```

Comment:
is bucket_id best here? or should this point to a specific asset? I see that there's <pdf_asset_file_name>, but can the bucket be derived from that?

Reply:
No, we need both bucket id and asset name